src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -971,7 +971,7 @@ public override object VisitField(FieldSymbol symbol, TypeCompilationState argum
                 }
 
                 // no need to emit the default ctor, we are not emitting those
-                if (methodSymbol.IsDefaultValueTypeConstructor(requireZeroInit: true))
+                if (methodSymbol.IsDefaultValueTypeConstructor(requireZeroInit: false))
```

Comment:
>IsDefaultValueTypeConstructor [](http://example.com/codeflow?start=33&length=29)

Since we no longer synthesize parameter-less struct constructors, it feels like the implementation of this method should change, Perhaps we even can drop the ```requireZeroInit``` parameter altogether.